data-raw/phenotypes.R

```diff
@@ -159,13 +161,198 @@ individual_metadata <- individual_metadata %>%
     mouse_model = as_factor(mouse_model)
   )
 
-## Combine data ----
+### Combine data ----
+
+phenotypes_5xfad <- phenotype_data %>%
+  inner_join(biospecimen_metadata, by = c("individual_id", "specimen_id")) %>%
+  inner_join(individual_metadata, by = "individual_id") %>%
+  select(individual_id, specimen_id, mouse_model, sex, age, tissue, phenotype, units, value)
```

Comment:
Whenever I see inner joins, I just want to make sure you intend on losing everything that's not intersecting between the two datasets.  If that's what you want, all clear! 💯

Reply:
Yup, as intended! I check if anything is missing / what's missing [here](https://github.com/Sage-Bionetworks/magora/blob/e2d839845d14cbe675ce79266aa0eba6ca9266ce/data-raw/phenotypes.R#L106), and we know that there are specimens not in the metadata, so want to inner join to only catch those with sufficient metadata to display.

Reply:
Lil sanity check!

Reply:
No problem, I appreciate it!